src/OpenSearch/Handlers/SigV4Handler.php

```diff
@@ -31,12 +33,13 @@ class SigV4Handler
      */
     public function __construct(
         string $region,
+        string $service,
```

Comment:
I know it's internal, but I would swap since signing service typically comes before the region of that service.